recipes-mender/mender/mender_0.1.bb

```diff
@@ -32,6 +32,7 @@ SYSTEMD_SERVICE_${PN} = "mender.service"
 FILES_${PN} += "${systemd_unitdir}/system/mender.service \
                 ${sysconfdir}/mender.conf \
                "
+SYSTEMD_AUTO_ENABLE = "disable"
```

Comment:
I just talked to @GregorioDiStefano, he's working around it in the tests, so it's alright to leave this out. The published image definitely needs to have Mender enabled, so I think that is what our guide should be.